l1-contracts/src.ts/deploy.ts

```diff
@@ -1259,6 +1260,30 @@ export class Deployer {
     }
   }
 
+  public async deployDAValidators(create2Salt: string, ethTxOptions: ethers.providers.TransactionRequest) {
+    ethTxOptions.gasLimit ??= 10_000_000;
+
+    // This address only makes sense on the L1, but we deploy it anyway to keep the script simple
+    const rollupDAValidatorAddress = await this.deployViaCreate2("RollupL1DAValidator", [], create2Salt, ethTxOptions);
+    console.log(`CONTRACTS_L1_ROLLUP_DA_VALIDATOR=${rollupDAValidatorAddress}`);
+
+    const validiumDAValidatorAddress = await this.deployViaCreate2(
+      "ValidiumL1DAValidator",
+      [],
+      create2Salt,
+      ethTxOptions
+    );
+    console.log(`CONTRACTS_L1_VALIDIUM_DA_VALIDATOR=${validiumDAValidatorAddress}`);
+
+    // This address only makes sense on the Sync Layer, but we deploy it anyway to keep the script simple
+    const relayedSLDAValidator = await this.deployViaCreate2("RelayedSLDAValidator", [], create2Salt, ethTxOptions);
+    console.log(`CONTRACTS_L1_RELAYED_SL_DA_VALIDATOR=${relayedSLDAValidator}`);
+
+    this.addresses.RollupL1DAValidator = rollupDAValidatorAddress;
+    this.addresses.ValidiumL1DAValidator = validiumDAValidatorAddress;
+    this.addresses.RelayedSLDAValidator = relayedSLDAValidator;
```

Comment:
Do I understand correctly the the relayedSLDAValidator is deployed on L1 as well? probably ok for now, but we will not want to keep it that way, no?

Reply:
```
    // This address only makes sense on the Sync Layer, but we deploy it anyway to keep the script simple
```

As said in the comment above. Yes, in the end we must remove it from l1